st2reactor/st2reactor/container/sensor_wrapper.py

```diff
@@ -61,6 +58,10 @@ def __init__(self, sensor_wrapper):
         self._sensor_wrapper = sensor_wrapper
         self._logger = self._sensor_wrapper._logger
         self._dispatcher = TriggerDispatcher(self._logger)
+        self._datastore_service = DatastoreService(logger=self._logger,
```

Comment:
Line 55 can be deleted ("DATASTORE_NAME_SEPARATOR")? It won't let me comment on that line since it's unchanged. :-(